### PR TITLE
AlexandrRaby_tgBot

### PR DESCRIPTION
**Pull request by Alexandr-R-94**

### CODE REVIEW

**Review by LiudmilaZhdanok — APPROVED** [review]
Lgfm

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

**Review by AnastasiaSheremet — APPROVED** [review]

File: homework/java/alexandr_raby/TelegramBot/TelegramBot/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>by.alexandrR</groupId>
+	<artifactId>TelegramBot</artifactId>
+	<version>0.0.1-SNAPSHOT_v.2.1</version>
```
Comment (max-java) [inline homework/java/alexandr_raby/TelegramBot/TelegramBot/pom.xml:13]: remove _v.2.1
Take a look at  
https://semver.org/lang/ru/
https://calver.org/

File: homework/java/alexandr_raby/TelegramBot/TelegramBot/pom.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	<name>TelegramBot</name>
+	<description>DenoTelegrabBotAlexandr</description>
```
Comment (max-java) [inline homework/java/alexandr_raby/TelegramBot/TelegramBot/pom.xml:15]: mistype?

File: homework/java/alexandr_raby/TelegramBot/TelegramBot/src/main/java/by/alexandrR/TelegramBot/service/MessageService.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                }else  if (magText.equalsIgnoreCase("/help")){
+                    return sendMessage.setText("Нужна моя помощь? Я готов помочь!");
+                }else if (magText.equalsIgnoreCase("Расскажи мне что-нибудь")) {
+                    return sendMessage.setText("Тебе рассказать о себе, или может быть о создании мира?");
+                } else if (magText.equalsIgnoreCase("Кто тебя создал?")) {
+                    return sendMessage.setText("Меня создал самый великий программер. Тот, чьё имя нельзя называть! :0");
+                }
+            }
+        }
+        return sendMessage.setText("Сейчас я тебя не понимаю. Напиши /start, /settings, /help либо задай правильный вопрос");
```
Comment (max-java) [inline homework/java/alexandr_raby/TelegramBot/TelegramBot/src/main/java/by/alexandrR/TelegramBot/service/MessageService.java:30]: Follow Single responsibility principle

**Review by max-java — APPROVED** [review]
lgfm

**Pull request merged**